LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/RawPayloadJsonConverter.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.NetworkServer
+{
+    using System;
+    using Newtonsoft.Json;
+
+    public class RawPayloadJsonConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType) => true;
```

Comment:
A comment would be good here.

Reply:
This was an experimentation that I forgot to remove. File has been deleted.